server.js

```diff
@@ -1,39 +0,0 @@
-const webpackDevMiddleware = require('webpack-dev-middleware');
```

Comment:
I'd rather use a custom server than the one provided by webpack. It allows more flexibility on configuration and for example you could use `internal-ip` package to expose the ip which is useful.